dist/colors.color-map.scss

```diff
@@ -1,90 +1,79 @@
 $polaris-colors: (
```

Comment:
Black and white are missing. Let me know if you need help - happy to pair on resolving it.

Reply:
Yea, pairing on this would be great. Thought I'd resolved the issue but things keep popping up missing :p